src/SearchButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
Excellent job extracting the search button. Good choice of name for your component and good to see you use PascalCase for your component and file names.